src/components/pagination/pagination.tsx

```diff
@@ -40,7 +40,7 @@ export class Pagination implements LocalizedComponent {
   //--------------------------------------------------------------------------
 
   /**
-   * When true, number values are displayed with a group separator corresponding to the language and country format.
+   * When `true`, number values are displayed with a group separator corresponding to the language and country format.
```

Comment:
I saw a different `groupSeparator` description changed above. Can we make them consistent? Your choice which we go with.